lib/livebook_web/live/output.ex

```diff
@@ -33,7 +35,8 @@ defmodule LivebookWeb.Output do
 
   defp border?({:stdout, _text}), do: true
   defp border?({:text, _text}), do: true
-  defp border?({:error, _message}), do: true
+  defp border?({:error, _message, {:missing_secret, _}}), do: false
```

Comment:
@josevalim do we want to glue these together?

![image](https://user-images.githubusercontent.com/17034772/187636749-77cd3503-393e-438b-ad7d-37160394d272.png)

Reply:
Glueing looks nice :+1:

Reply:
So maybe it is better to send the missing secret as a separate output so we get the gluing for free?

Reply:
I think we can just render it as a single output and add the line separator. We can escape the box padding with negative margin somewhere :)